dashboard/db/migrate/20241025182954_update_census_summary_school_year_name.rb

```diff
@@ -0,0 +1,5 @@
+class UpdateCensusSummarySchoolYearName < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :census_summaries, :school_year, :access_report_year
```

Comment:
I'm curious if this also needs to drop the old index and apply the new one, or does ActiveRecord take care of this for us?

Reply:
here's an old migration that drops an old index and adds a new one https://github.com/code-dot-org/code-dot-org/blob/543bcc8d8ca04bd6996373cc3caddf3776fdf79a/dashboard/db/migrate/20131212190308_unmodify_followers_index.rb

Reply:
I originally did the drop/add but saw that without it, ActiveRecord [took care of it](https://github.com/code-dot-org/code-dot-org/pull/62030/files#diff-962f2f608ae3ac7ce84f1a84a6a43f4f14eada4d98fd5530c0b0905555a6ffafL15) for me!

Reply:
more rails magic!! 🪄 

could you maybe add a screenshot? `show index from census_summaries;`

Reply:
ooohhh, is `schema.rb` a generated file? I get it now 😁

Reply:
Yep yep!